core/state_transition.go

```diff
@@ -76,10 +76,10 @@ type Message interface {
 }
 
 // IntrinsicGas computes the 'intrinsic gas' for a message with the given data.
-func IntrinsicGas(data []byte, contractCreation, homestead bool) (uint64, error) {
+func IntrinsicGas(data []byte, contractCreation, eip155Active bool, eip2028Active bool) (uint64, error) {
```

Comment:
You are going to hate me now, but `ChainConfig` has `IsIstanbul`, and `chainRules` have `IsEIPXYZ`. Lets keep it consistent and name these `isEIP155` and `isEIP2028` instead of starting a new style with `Active`. @holiman ?

Reply:
`isEIP2028` and the like is fine with me

Reply:
I update the arguments to `isEIP` style.